packages/BetterPhpDocParser/PhpDocInfo/PhpDocInfo.php

```diff
@@ -189,11 +189,6 @@ public function hasByName(string $name): bool
         return (bool) $this->getTagsByName($name);
     }
 
-    public function getByName(string $name): ?Node
```

Comment:
I think you can mark this as `@api` so it can be keep as `getByName()` seems usefull to be kept

Reply:
Thanks!